From 544fa712f2a894622a7dcf610a1d23edb66e6a44 Mon Sep 17 00:00:00 2001
From: Xiaoliang Yang <xiaoliang.yang_1@nxp.com>
Date: Tue, 9 Apr 2019 14:18:59 +0800
Subject: [PATCH 688/706] mscc: switch: fix checkpatch issues

Remove check patch issues from TSN-switch-configuration-param-get.patch and
TSN-switch-fix-cbstreamidset-issue.patch.

Signed-off-by: Xiaoliang Yang <xiaoliang.yang_1@nxp.com>
(cherry picked from commit e816cdd4e086b3b2a2626161fc45abb80d9f7606)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/net/ethernet/mscc/tsn_switch.c | 23 +++++++++++++++--------
 net/tsn/genl_tsn.c                     |  2 --
 2 files changed, 15 insertions(+), 10 deletions(-)

diff --git a/drivers/net/ethernet/mscc/tsn_switch.c b/drivers/net/ethernet/mscc/tsn_switch.c
index be2015188f7b..4fae48e05b8e 100644
--- a/drivers/net/ethernet/mscc/tsn_switch.c
+++ b/drivers/net/ethernet/mscc/tsn_switch.c
@@ -501,7 +501,8 @@ int switch_cb_streamid_get(struct net_device *ndev, u32 index,
 
 	/*READ command MACACCESS.VALID(11 bit) must be 0 */
 	ocelot_write(ocelot,
-		     ANA_TABLES_MACACCESS_MAC_TABLE_CMD(MACACCESS_CMD_READ) ,
+		     ANA_TABLES_MACACCESS_MAC_TABLE_CMD(
+			    MACACCESS_CMD_READ),
 		     ANA_TABLES_MACACCESS);
 
 	val = ocelot_read(ocelot, ANA_TABLES_MACACCESS);
@@ -564,7 +565,7 @@ int switch_cb_streamid_set(struct net_device *ndev, u32 index, bool enable,
 	if (streamid->type != 1)
 		return -EINVAL;
 
-	if (enable == TRUE) {
+	if (enable) {
 		netdev_dbg(ndev, "index=%d mac=0x%llx vid=0x%x sfid=%d dst=%d\n",
 			   index, streamid->para.nid.dmac,
 			   streamid->para.nid.vid,
@@ -593,13 +594,15 @@ int switch_cb_streamid_set(struct net_device *ndev, u32 index, bool enable,
 		ocelot_write(ocelot, ANA_TABLES_MACACCESS_VALID |
 			     ANA_TABLES_MACACCESS_ENTRYTYPE(1) |
 			     ANA_TABLES_MACACCESS_DEST_IDX(dst_idx) |
-			     ANA_TABLES_MACACCESS_MAC_TABLE_CMD(MACACCESS_CMD_WRITE),
+			     ANA_TABLES_MACACCESS_MAC_TABLE_CMD(
+				    MACACCESS_CMD_WRITE),
 			     ANA_TABLES_MACACCESS);
 
 		ocelot_write(ocelot, ANA_TABLES_MACACCESS_VALID |
 			     ANA_TABLES_MACACCESS_ENTRYTYPE(1) |
 			     ANA_TABLES_MACACCESS_DEST_IDX(dst_idx) |
-			     ANA_TABLES_MACACCESS_MAC_TABLE_CMD(MACACCESS_CMD_LEARN),
+			     ANA_TABLES_MACACCESS_MAC_TABLE_CMD(
+				    MACACCESS_CMD_LEARN),
 			     ANA_TABLES_MACACCESS);
 	} else {
 		netdev_info(ndev, "disable stream set\n");
@@ -613,7 +616,8 @@ int switch_cb_streamid_set(struct net_device *ndev, u32 index, bool enable,
 			     ANA_TABLES_MACHDATA);
 
 		ocelot_write(ocelot,
-			     ANA_TABLES_MACACCESS_MAC_TABLE_CMD(MACACCESS_CMD_FORGET),
+			     ANA_TABLES_MACACCESS_MAC_TABLE_CMD(
+				    MACACCESS_CMD_FORGET),
 			     ANA_TABLES_MACACCESS);
 	}
 
@@ -1051,7 +1055,8 @@ int switch_seq_gen_set(struct net_device *ndev, u32 index,
 	ocelot_write(ocelot,
 		     ANA_TABLES_STREAMACCESS_GEN_REC_SEQ_NUM(seq_num) |
 		     ANA_TABLES_STREAMACCESS_SEQ_GEN_REC_ENA |
-		     ANA_TABLES_STREAMACCESS_STREAM_TBL_CMD(SFIDACCESS_CMD_WRITE),
+		     ANA_TABLES_STREAMACCESS_STREAM_TBL_CMD(
+			    SFIDACCESS_CMD_WRITE),
 		     ANA_TABLES_STREAMACCESS);
 
 	return 0;
@@ -1086,7 +1091,8 @@ int switch_seq_rec_set(struct net_device *ndev, u32 index,
 	ocelot_write(ocelot,
 		     ANA_TABLES_STREAMACCESS_SEQ_GEN_REC_ENA |
 		     ANA_TABLES_STREAMACCESS_GEN_REC_TYPE |
-		     ANA_TABLES_STREAMACCESS_STREAM_TBL_CMD(SFIDACCESS_CMD_WRITE),
+		     ANA_TABLES_STREAMACCESS_STREAM_TBL_CMD(
+			    SFIDACCESS_CMD_WRITE),
 		     ANA_TABLES_STREAMACCESS);
 
 	return 0;
@@ -1103,7 +1109,8 @@ int switch_cb_get(struct net_device *ndev, u32 index, struct tsn_cb_status *c)
 		     ANA_TABLES_STREAMTIDX);
 
 	ocelot_write(ocelot,
-		     ANA_TABLES_STREAMACCESS_STREAM_TBL_CMD(SFIDACCESS_CMD_READ),
+		     ANA_TABLES_STREAMACCESS_STREAM_TBL_CMD(
+			    SFIDACCESS_CMD_READ),
 		     ANA_TABLES_STREAMACCESS);
 
 	val = ocelot_read(ocelot, ANA_TABLES_STREAMACCESS);
diff --git a/net/tsn/genl_tsn.c b/net/tsn/genl_tsn.c
index 6d671ee10279..8c245201f840 100644
--- a/net/tsn/genl_tsn.c
+++ b/net/tsn/genl_tsn.c
@@ -3274,8 +3274,6 @@ static const struct genl_ops tsnnl_ops[] = {
 		.policy		= tsn_cmd_policy,
 		.flags		= GENL_ADMIN_PERM,
 	},
-
-
 	{
 		.cmd		= TSN_CMD_PCPMAP_SET,
 		.doit		= tsn_pcpmap_set,
-- 
2.17.1

